content/en/docs/Concepts/crds/clusterserviceversion.md

```diff
@@ -96,6 +96,7 @@ spec:
     # name is the metadata.name of the CRD (which is of the form <plural>.<group>)
     - name: memcacheds.cache.example.com
       # version is the spec.versions[].name value defined in the CRD
+      # while version is not required, semver template functionality relies on its presence
```

Comment:
I took a look at #311 and it looks like it is referring to the need for the `spec.version` field (https://github.com/operator-framework/olm-docs/pull/319/files#diff-0fd678129cce2ee24610d34be80b5bd6aaef3f7cc1191123eafb86e732b76dbcR45) to be present for the semver template. This PR, in it's current form, is stating that the CRD version is required and doesn't resolve #311 .

Additionally it might be helpful to add a note to https://olm.operatorframework.io/docs/reference/catalog-templates/#semver-template to mention the requirement for the CSV's `spec.version` field to be populated where folks may go back to reference information on the semver template or first learn about it.

Reply:
WDYT of the following?
```suggestion
      # spec.version is an optional field.
      # the spec.version field is required if you want to use the semver template for your catalog.
```

Reply:
I'm a little reluctant to update CSV docs at because:
1. if a user has no intention to order by spec.version, this is just noise.
2. in this location it feels common-sense.  "What do you mean that to sequence I have to provide a value?" feels like a question that has an obvious answer.
3. the reliance on spec.version actually lies in semver catalog template, which is alpha software, so it feels awkward allowing alpha software to impose any burden on core implementation w/o the usual gradual adoption workflow.

I think it makes more sense to put the caveat in [semver template](https://olm.operatorframework.io/docs/reference/catalog-templates/#semver-template) area in line with [#311's note](https://github.com/operator-framework/olm-docs/issues/311#issue-1907782176).  Ideally that would also include a reference to the CSV location to show the field in question.